packages/core/src/reducers/index.ts

```diff
@@ -78,20 +76,6 @@ export {
 };
 export { JsonFormsCore, ValidationMode };
 
-export const jsonformsReducer = (
-  additionalReducers = {}
-): Reducer<JsonFormsSubStates> =>
-  combineReducers<JsonFormsSubStates>({
-    core: coreReducer,
-    renderers: rendererReducer,
-    cells: cellReducer,
-    config: configReducer,
-    uischemas: uischemaRegistryReducer,
-    defaultData: defaultDataReducer,
-    i18n: i18nReducer,
-    ...additionalReducers
-  });
-
```

Comment:
We need to keep the `jsonformsReducer` as they are needed when using JSON Forms with Redux, see for example the React seed or the example apps.